BEIMA.Client/src/pages/Devices/DevicePage.js

```diff
@@ -15,7 +15,7 @@ const DevicePage = () => {
   const [loading, setLoading] = useState(true);
   const [device, setDevice] = useState(null);
   const [image, setImage] = useState(null)
-  const [documents, setDocuments] = useState([])
+  const [documents, setDocuments] = useState({})
```

Comment:
This should still be an array as when it gets set by setDocuments(device.files), it'll become an array again as device.files is an array of json objects.

Reply:
Good catch, thanks! I'm not sure how it is working now, but I'll play around with it

Reply:
Changed back to array